src/desktop/game_mode.rs

```diff
@@ -154,7 +154,7 @@ impl<'a> GameMode<'a> {
     /// See also [`RegisterGame`](https://flatpak.github.io/xdg-desktop-portal/docs/doc-org.freedesktop.portal.GameMode.html#org-freedesktop-portal-gamemode-registergame).
     #[doc(alias = "RegisterGame")]
     pub async fn register(&self, pid: u32) -> Result<(), Error> {
-        let status = self.0.call("RegisterGame", &(pid)).await?;
+        let status = self.0.call("RegisterGame", &(pid as i32)).await?;
```

Comment:
why not replace the function signature? is this in order to have a backportable variant?

Reply:
Originally, I had two reasons not to change the signature:
1. Backward compatibility, in order not to break code that already uses `u32`. (Now writing this I realized that probably no one uses it as it doesn't work...)
2. Better interoperability with Rust, as Rust uses `u32` for pids: https://doc.rust-lang.org/std/process/fn.id.html, https://doc.rust-lang.org/std/process/struct.Child.html#method.id

Reply:
Right, in such case I would add 

```
// Matches the type used in std
pub type Pid = u32;
```

and pass that to the functions instead.

Reply:
Sure thing, added the `Pid` type alias and replaced most pids in API. There are instances where `u64` is used though, but I haven't touched them.

Let me know if that's what you meant.